packages/backend-app-api/src/services/implementations/httpRouter/createCredentialsBarrier.ts

```diff
@@ -73,12 +73,15 @@ export function createCredentialsBarrier(options: {
       predicate(req.path),
     );
 
-    await httpAuth.credentials(req, {
-      allow: ['user', 'service'],
-      allowedAuthMethods: allowsCookie ? ['token', 'cookie'] : ['token'],
-    });
-
-    next();
+    httpAuth
+      .credentials(req, {
```

Comment:
i know it's safe the way we implement this, but it always scares me to write these calls "naked" in a sync context since we don't know that a replacement httpAuth implementation doesn't synchronously throw...